package-managers/qubes-download-dom0-updates.sh

```diff
@@ -72,15 +72,15 @@ fi
 
 mkdir -p $DOM0_UPDATES_DIR/etc
 
-if [ -e /etc/debian_version ]; then
-    # Default rpm configuration on Debian uses ~/.rpmdb for rpm database (as
-    # rpm isn't native package manager there)
-    mkdir -p "$DOM0_UPDATES_DIR$HOME"
-    rm -rf "$DOM0_UPDATES_DIR$HOME/.rpmdb"
-    cp -r "$DOM0_UPDATES_DIR/var/lib/rpm" "$DOM0_UPDATES_DIR$HOME/.rpmdb"
+# Check if we need to copy rpmdb somewhere else
+DBPATH=`rpm -E '%{_dbpath}'`
```

Comment:
```suggestion
DBPATH=$(rpm --eval '%{_dbpath}')
```

shellcheck complains about the use of backticks, and `--eval` is more readable than `-E`.